src/main/java/alexandr_samohvalov/ConditionsMain.java

```diff
@@ -0,0 +1,62 @@
+package alexandr_samohvalov;
+
+import java.util.Scanner;
+
+public class ConditionsMain {
+    // Check if a number is positive, negative, or zero.
+    public static void checkNumType(String[] CheckNumType) {
+        Scanner scanner = new Scanner(System.in);
+        int a = scanner.nextInt();
+        if (a > 0) {
+            System.out.println("Number is positive");
+        } else if (a < 0) {
+            System.out.println("Number is negative");
+        } else {
+            System.out.println("Number is equal to zero");
+        }
+    }
+
+    //Check if a number is divisible by both 2 and 3.
+    public static void numDivisibleBy2and3(String[] NumDivisibleBy2and3) {
+        System.out.println("Check if number is divided by 2 and 3");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Introduce number");
+        int a = scanner.nextInt();
+        int b = 2;
+        int c = 3;
+        if (a % b == 0 && a % c == 0) {
+            System.out.println("Number can be divided by 2 and 3");
+        } else if (a % b == 0) {
+            System.out.println("Number can be divided by 2");
+        } else if (a % c == 0) {
+            System.out.println("Number can be divided by 3");
+        } else {
+            System.out.println("Number can not be divided by 2 and 3 ");
+        }
+    }
+
+
+// Determine the grade based on a given score.
+
+    public static void gradeBaseScore(String[] GradeBaseScore) {
```

Comment:
1. All parameters and variables should start from lowercase letter
2. you don''t use GradeBaseScore inside method, so please remove it